bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -5158,7 +5158,7 @@ spec:
     - email: container.storage.modules@dell.com
       name: Dell Container Storage Modules
   maturity: stable
-  minKubeVersion: 1.28.0
+  minKubeVersion: 1.24.0
```

Comment:
Why are we downgrading min kube version?

Reply:
@rajendraindukuri I think minkuberversion is updated based on the manifests generated based on the latest operator-sdk. In our case, we need to mention the lowest kubernetes version supported across all orchestrators we are claiming support. Based on that ... the latest MKE/RKE version we have in support matrix is on k8s 1.24.. Thats the reason we are updating this to 1.24 from last few releases manually.